[2024-04-08T03:56:24.787+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: real_estates.extract_real_estates manual__2024-04-08T03:55:40.228975+00:00 [queued]>
[2024-04-08T03:56:24.807+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: real_estates.extract_real_estates manual__2024-04-08T03:55:40.228975+00:00 [queued]>
[2024-04-08T03:56:24.808+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-04-08T03:56:24.838+0000] {taskinstance.py:2217} INFO - Executing <Task(_PythonDecoratedOperator): extract_real_estates> on 2024-04-08 03:55:40.228975+00:00
[2024-04-08T03:56:24.857+0000] {standard_task_runner.py:60} INFO - Started process 78214 to run task
[2024-04-08T03:56:24.864+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'real_estates', 'extract_real_estates', 'manual__2024-04-08T03:55:40.228975+00:00', '--job-id', '302', '--raw', '--subdir', 'DAGS_FOLDER/real_estates/__init__.py', '--cfg-path', '/tmp/tmpw12awz5z']
[2024-04-08T03:56:24.870+0000] {standard_task_runner.py:88} INFO - Job 302: Subtask extract_real_estates
[2024-04-08T03:56:24.971+0000] {task_command.py:423} INFO - Running <TaskInstance: real_estates.extract_real_estates manual__2024-04-08T03:55:40.228975+00:00 [running]> on host 05a350db99a0
[2024-04-08T03:56:25.479+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='real_estates' AIRFLOW_CTX_TASK_ID='extract_real_estates' AIRFLOW_CTX_EXECUTION_DATE='2024-04-08T03:55:40.228975+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-08T03:55:40.228975+00:00'
[2024-04-08T03:56:28.098+0000] {real_estates.py:37} INFO - Batch 1/1 fetched 121 pages
[2024-04-08T03:56:50.194+0000] {local_task_job_runner.py:302} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2024-04-08T03:56:50.199+0000] {process_utils.py:131} INFO - Sending 15 to group 78214. PIDs of all processes in the group: [78214]
[2024-04-08T03:56:50.200+0000] {process_utils.py:86} INFO - Sending the signal 15 to group 78214
[2024-04-08T03:56:50.201+0000] {taskinstance.py:2483} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-04-08T03:56:50.333+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=78214, status='terminated', exitcode=0, started='03:56:23') (78214) terminated with exit code 0
